woocommerce/payment-gateway/class-sv-wc-payment-gateway.php

```diff
@@ -509,13 +512,13 @@ protected function get_payment_form_js_localized_script_params() {
 	protected function enqueue_gateway_assets() {
 
 		$handle = $this->get_gateway_js_handle();
-		$js_path   = $this->get_plugin()->get_plugin_path() . '/assets/js/frontend/' . $handle . '.min.js';
+		$js_path   = $this->get_plugin()->get_plugin_path() . '/assets/dist/frontend/' . $handle . '.js';
```

Comment:
Not sure I understood this one. We are not changing the path for the plugin JS files, just the gateway. So I don't think we will have a `$this->get_plugin()->get_plugin_path() . '/assets/dist/frontend/'` folder.

Reply:
>So I don't think we will have a $this->get_plugin()->get_plugin_path() . '/assets/dist/frontend/' folder.

@deivamagalhaes yep that's right. that dir doesn't exist in the plugins, so their JS won't be enqueued properly. While this method is in the gateway class, it applies somewhat at a plugin level unless overridden by individual gateways. Most of them have a single plugin-level JS file that is used by all of the gateways that it has.

Using Cybersource as an example, its main JS file is `woocommerce-gateway-cybersource/assets/js/frontend/wc-cybersource.min.js`. The plugin itself doesn't enqueue this file - the framework does that for it in this method.

With this change, the framework will now look for that JS file at `woocommerce-gateway-cybersource/assets/dist/frontend/wc-cybersource.js` and won't find it because plugins haven't adopted the framework's new compiling structure. On the accompanying Cybersource branch, I can see that the JS for the flex form is no longer enqueued.

So the `$js_path` and `$js_url` values need to remain unchanged to maintain backwards compatibility with how plugins have their JS structured currently. Does that make sense?

Reply:
I thought you were asking me to change that line, instead of telling me not to change it. 🤦‍♀️  

We should also keep the `.min` suffix, right?

Reply:
Fixed in https://github.com/skyverge/wc-plugin-framework/pull/507/commits/3cba64529d90c7eccd98c495c7546dba563524c3.